Editor/AGS.Editor/DataFileWriter.cs

```diff
@@ -64,7 +64,6 @@ private static byte[] GetAnsiBytes(string text, int min_size, int max_size, int
             //
             // The output array will be at least min_size in size.
             byte[] bytes = new byte[Math.Max(min_size, Encoding.Default.GetByteCount(text) + reserve_bytes)];
-            int data_length = Encoding.Default.GetBytes(text, 0, text.Length, bytes, 0);
             if (max_size <= 0 || bytes.Length <= max_size)
```

Comment:
This is not right, the call to GetBytes fills in bytes array, that's the core functionality of the function.

Why data_length is not used is a curious case, may be a mistake. I will look into this.